### PR TITLE
solidity core concepts

### PR DESCRIPTION
**Pull request by hellomohitsangwan**
In this PR I have added some basic things which you need to know in order to start solidity and blockchain development

### CODE REVIEW

File: Beginner_concepts/basic_data_types.sol [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    bool public is18 = true;
+    int public age = -18;    //a int can be signed(+ve or -vs but a uint can only be +ve)
+    uint public profit = 18;  
+    // uint8 public profit = 18;  
+     uint256 public profit = 18;  
+    
+    constructor() public {
+        name = "mohit";
+    }
+    // function get() public view returns(string memory) {
```
Comment (iaraGkitarP) [inline Beginner_concepts/basic_data_types.sol:17]: remove useless pieces of code

File: Beginner_concepts/enums.sol [hunk 2]
```diff
@@ -0,0 +1,22 @@
+pragma solidity ^0.6.7;
+
+contract enumTut {
+        //enums
```
Comment (iaraGkitarP) [inline Beginner_concepts/enums.sol:4]: indentation of comment, maybe use a few blank lines for readablility

**Review by iaraGkitarP — CHANGES_REQUESTED** [review]

File: Beginner_concepts/basic_data_types.sol (hunk 1 above)
Comment (hellomohitsangwan) [reply Beginner_concepts/basic_data_types.sol:17]: okay, i'll refactor

File: Beginner_concepts/enums.sol (hunk 2 above)
Comment (hellomohitsangwan) [reply Beginner_concepts/enums.sol:4]: okay,i'll take care of that futurere

**Comment by iaraGkitarP** [comment]
Looks good to go

**Pull request merged**